src/services/NodeMonitor.ts

```diff
@@ -166,6 +166,13 @@ export class NodeMonitor {
 			}
 
 			if (isAPIRole(node.roles)) {
+				if (node.version === 0) {
+					const nodeInfo = await ApiNodeService.getNodeInfo(node.host, 3000, 'http:');
+
+					if (nodeInfo) {
+						nodeWithInfo.version = nodeInfo.version;
+					}
+				}
```

Comment:
Why do we need to call node/info with HTTP(3000) here, looked redundant to me? 

Instead of using this block, can we use the result of the following `await ApiNodeService.getStatus(node.host)` in line 176? Because we already make a `node/info` call inside `ApiNodeService.getStatus`, no?

Reply:
Yes, I was thinking the same thing.

but it required to add new properties (version) in `interface ApiStatus`.

![Screenshot 2021-11-20 at 8 59 15 PM](https://user-images.githubusercontent.com/5649156/142727402-7f77def9-5941-4b0f-bc68-61442856b026.png)

Do you think will be confusing?